httpgrpc/server/server.go

```diff
@@ -165,8 +165,8 @@ func NewClient(address string) (*Client, error) {
 	}, nil
 }
 
-// HTTPRequest wraps an ordinary HTTPRequest with a gRPC one
-func HTTPRequest(r *http.Request) (*httpgrpc.HTTPRequest, error) {
+// WrapHTTPRequest wraps an ordinary http.Request up into an httpgrpc.HTTPRequest
+func WrapHTTPRequest(r *http.Request) (*httpgrpc.HTTPRequest, error) {
```

Comment:
there is one usage of this in Mimir/GEM that would need updated with the new name, but if we're super adverse to changing that, it could remain as it was.

Reply:
It's not "wrapping" (new message doesn't keep original request), but full conversion, so the new naming is wrong.